examples/acados_matlab_octave/getting_started/minimal_example_closed_loop.m

```diff
@@ -28,146 +28,130 @@
 % POSSIBILITY OF SUCH DAMAGE.;
 
 
-
-% NOTE: `acados` currently supports both an old MATLAB/Octave interface (< v0.4.0)
-% as well as a new interface (>= v0.4.0).
-
-% THIS EXAMPLE still uses the OLD interface. If you are new to `acados` please start
-% with the examples that have been ported to the new interface already.
-% see https://github.com/acados/acados/issues/1196#issuecomment-2311822122)
-
 clear all; clc;
 
-model_path = fullfile(pwd,'..','pendulum_on_cart_model');
-addpath(model_path)
-
 check_acados_requirements()
 
 % initial state
 x0 = [0; 0; 0; 0];  % start at stable position
 
+%% SOLVER OPTIONS
+solver_options = AcadosOcpOptions();
```

Comment:
In the Python examples, we typically do create `ocp = AcadosOcp()`  and then set options:
`ocp.solver_options.* = ...`
I somehow prefer this a bit.
Also maybe we can check if you can overwrite `ocp.solver_options` by something that is not an `AcadosOcpOptions` object.

Reply:
but somehow we don't do it for the model :shrug:

Reply:
Right, it is somehow more common to have a function that sets up the model. But I think it is nice that the example matches the python one closely now.